Atlas Edit Proposal — 2026-06-08 (#255)
makerdao/next-gen-atlas: content/A/1/10/2/4/12/4/6/document.md

###### A.1.10.2.4.12.4.6 - Secondary Reporting Counterparty [Core]

> If a validator suspects that the Core Facilitator is malicious or compromised, they must notify Sky Governance by reporting the issues they found publicly. This permits the Sky Governance community to independently investigate the issue and act together to mitigate an attack.

Reviewer: Content analysis of amendments regarding logical consistency, systemic loopholes, and linguistics: [Notion](https://sky-proposal-analysis.notion.site/Atlas-Edit-Weekly-Cycle-Proposal-2026-06-15-37ea66bbb6b8802281e5cfa68ee5a1a8).

```suggestion
If a validator suspects that Core Facilitators are malicious or colluding, they must notify Sky Governance by reporting the issues they found publicly. This permits the Sky Governance community to independently investigate the issue and act together to mitigate an attack.
```
